WEB_ROOT/UTILITIES/API_UTILITIIES/apiDetailsProvider.js

```diff
@@ -57,19 +59,21 @@ export const API_PAYLOAD_FACTORY = {
 
     },
     ADVERTISER: {
-        LOGIN: (email, password) => ({
-            usernameOrEmail:email,
+ feature/api-login
+        LOGIN: (usernameOrEmail, password) => ({
+            usernameOrEmail:usernameOrEmail,
             password:password
         }),
 
-        CREATE: (FirstName, LastName,UserName,PhoneNumber,EmailAddress,Password,ConfirmPassword) => ({
-            firstName:FirstName,
-            lastName:LastName,
-            email:EmailAddress,
-            username:UserName,
-            phoneNumber:PhoneNumber,
-            password:Password,
-            confirmPassword:ConfirmPassword,
+        REGISTER: (advertiserFirstName, advertiserLastName,advertiserUserName,advertiserPhoneNumber,advertiserEmailAddress,advertiserStatus,advertiserPassword,advertiserConfirmPassword) => ({
```

Comment:
The changes made here will make the register endpoints fail . Address and correct where necessary